Add new divine testnet chain into registry

kctoptal opened the pull request.

kws1207 commented on cosmos/divine-testnet.json:
```diff
@@ -0,0 +1,53 @@
+{
+  "chainId": "divine",
```
Please ensure the `chainId` matches the filename. (e.g., `divine-testnet` or `divine-testnet-1`)

https://github.com/chainapsis/keplr-chain-registry/actions/runs/20031502150/job/57511029065

kws1207 reviewed (changes requested):
The RPC endpoint returns a different chainId than configured. Please verify the correct chainId from the chain and update accordingly (both the chainId field and the filename).

https://github.com/chainapsis/keplr-chain-registry/actions/runs/20053169598/job/57513982277

kctoptal commented:
> The RPC endpoint returns a different chainId than configured. Please verify the correct chainId from the chain and update accordingly (both the chainId field and the filename).
> 
> https://github.com/chainapsis/keplr-chain-registry/actions/runs/20053169598/job/57513982277

Fixed the chain id issue. RPC chain id is now changed. Now file mentioned chain id will be returned by RPC url as well. Thanks.
